README.md

```diff
@@ -91,6 +93,11 @@ CaBot v2 uses ROS1, ROS2, and ros1_bridge to use [navigation2](https://github.co
                 # if there is no nvidia-smi and config name is not set, automatically set to 'nuc'
     -3          # equivalent to '-c rs3'
   ```
+- (optional) Run the gnss container before running launch.sh if you use a gnss receiver
+  ```
+  docker-compose -f docker-compose-gnss.yaml up
+  ```
```

Comment:
Added the usage of docker-compose-gnss.yaml file to README.

Reply:
GNSS container is used to send RTK correction information to a GNSS receiver connected to the computer.